<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiscImageCreator GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2" colspan="7">
     <widget class="QComboBox" name="cb_driveLetter">
      <property name="toolTip">
       <string>Select an optical drive to read from.</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="8">
     <widget class="QCheckBox" name="zipFiles">
      <property name="toolTip">
       <string>Zip all the files relevant for logging the dump's properties.</string>
      </property>
      <property name="text">
       <string>Zip log files when finished</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="8">
     <widget class="Line" name="line_upper">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1" alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label_fileName">
      <property name="text">
       <string>Output File Name</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_discType">
      <property name="text">
       <string>Disc Type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_dir">
      <property name="text">
       <string>Output Directory</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QRadioButton" name="rb_speed16">
      <property name="toolTip">
       <string>Read disc at 16 times speed.</string>
      </property>
      <property name="text">
       <string>16</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QRadioButton" name="rb_speed4">
      <property name="toolTip">
       <string>Read disc at 4 times speed.</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="6">
     <widget class="QLineEdit" name="le_dir">
      <property name="toolTip">
       <string>The disc image will be stored in this directory.</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="8">
     <widget class="QPlainTextEdit" name="pt_console">
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="7" colspan="2">
     <widget class="QPushButton" name="pb_start">
      <property name="toolTip">
       <string>Start</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_drive">
      <property name="text">
       <string>Drive</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QPushButton" name="pb_browseDir">
      <property name="toolTip">
       <string>Browse</string>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="7">
     <widget class="QComboBox" name="cb_discType">
      <property name="toolTip">
       <string>What type of disc will be imaged?</string>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAlphabetically</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_driveSpeed">
      <property name="text">
       <string>Drive Speed</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="8">
     <widget class="Line" name="line_lower">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QRadioButton" name="rb_speed8">
      <property name="toolTip">
       <string>Read disc at 8 times speed.</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QRadioButton" name="rb_speed48">
      <property name="toolTip">
       <string>Read disc at 48 times speed.</string>
      </property>
      <property name="text">
       <string>48</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="7">
     <widget class="QLineEdit" name="le_fileName">
      <property name="toolTip">
       <string>Give the image read from the disc a name.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QRadioButton" name="rb_custom">
      <property name="toolTip">
       <string>Read disc at a custom speed.</string>
      </property>
      <property name="text">
       <string>Custom...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7" colspan="2">
     <widget class="QLineEdit" name="le_customDriveSpeed">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Enter a custom disc read speed.</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections/>
</ui>
